v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Disaster Recovery
+summary: Learn how about CockrachDB disaster recovery capabilities and what to do if you encounter an issue.
+toc: true
+---
+
+While CockroachDB is built to be fault-tolerant and to recover automatically, sometimes disasters happen. A _disaster_ is any event that puts your cluster at risk, and usually means your cluster is experiencing [hardware failure](#hardware-failure), [data failure](#data-corruption), or has [compromised security keys](#compromised-security-keys). Having a disaster recovery plan enables you to recover quickly, while limiting the consequences.
+
+## Hardware failure
+
+When planning to survive hardware failures, these are the minimum replication factors to apply in order to have greater resilience.
```

Comment:
What section will this live under? I ask because are we telling users how to survive faults? Or are we explaining to them the things to do to be fault tolerant?